openpype/hosts/houdini/api/plugin.py

```diff
@@ -316,6 +324,11 @@ def get_network_categories(self):
     def apply_settings(self, project_settings):
         """Method called on initialization of plugin to apply settings."""
 
+        # Apply General Settings
+        self.add_publish_button = \
+            project_settings["houdini"]["general"]["add_self_publish_button"]
```

Comment:
Be carefull about new keys in settings...
```suggestion
        houdini_general_settings = project_settings["houdini"]["general"]
        self.add_publish_button = houdini_general_settings.get(
            "add_self_publish_button", False)
```